src/forms/progressive-profiling-popup/index.jsx

```diff
@@ -53,6 +53,7 @@ const ProgressiveProfilingForm = () => {
 
   const countryCookieValue = getCountryCookieValue();
   const countryList = useMemo(() => getCountryList(getLocale()), []);
+  const showRecommendations = getConfig().SHOW_RECOMMENDATIONS;
```

Comment:
we should do feature parity with authn, we don't show recommendations if the user is from enrollment flow or the user has next URL. 

i can't remember the exact use case with next query param but we should definitely check this.